test/bench/custom/benchmarks.mjs

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable strict */
+import { BSON } from '../../../lib/bson.mjs';
+
+const ObjectId_isValid = [
+  function objectid_isvalid_strlen() {
+    BSON.ObjectId.isValid('a');
+  },
+  function objectid_isvalid_bestcase_false() {
```

Comment:
what makes this the best case? and the other the worst case?

Reply:
Most or least short-circuiting. Updated the test names and added comments to clarify!

Reply:
Test case names lgtm now - thanks!